lib/web_ui/lib/src/engine/surface/recording_canvas.dart

```diff
@@ -183,10 +183,12 @@ class RecordingCanvas {
   }
 
   void drawColor(ui.Color color, ui.BlendMode blendMode) {
-    _hasArbitraryPaint = true;
-    _didDraw = true;
-    _paintBounds.grow(_paintBounds.maxPaintBounds);
-    _commands.add(PaintDrawColor(color, blendMode));
+    drawRect(
+        _paintBounds.maxPaintBounds,
```

Comment:
We can get `NaN` in `maxPaintBounds` (see https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/surface/recording_canvas.dart#L1887)

Reply:
reverted change.